sopel/formatting.py

```diff
@@ -94,8 +94,9 @@ def color(text, fg=None, bg=None):
     """Return the text, with the given colors applied in IRC formatting.
 
     The color can be a string of the color name, or an integer between 0 and
-    99. The known color names can be found in the `colors` class of this
-    module."""
+    99. The known color names can be found in the :class:`~.colors` class of
```

Comment:
@Exirel should I leave this dot out, then?

```suggestion
    99. The known color names can be found in the :class:`~colors` class of
```

Reply:
I think you can, yes. The best way to know is to run Sphinx, to be sure!